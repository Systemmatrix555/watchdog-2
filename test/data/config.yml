contact: DSIT DevOps <dsit-devops@edf.fr>
docs_link: https://groupe-edf.github.io/watchdog/docs/
error_message_prefix: "GL-HOOK-ERR: "
logs_level: info
logs_format: json
logs_path: watchdog.log
concurrent: 4
output_format: text
security:
  reveal_secrets: 4
