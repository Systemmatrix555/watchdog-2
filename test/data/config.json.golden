{
  "_comment": "Docker config.json file",
  "auths": {
    "https://index.docker.io/v1/": {
      "email": "john.doe@acme.com",
      "auth":"X3Rva2VuOjEyMzQ1Njc4OTBBQkNERUY="
    }
  }
}
